seed/views/v3/users.py

```diff
@@ -0,0 +1,886 @@
+)
+from seed.utils.api import api_endpoint_class
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.organizations import create_organization
+from rest_framework.status import HTTP_400_BAD_REQUEST
+
+_log = logging.getLogger(__name__)
+
+
+def _get_js_role(role):
+    """return the JS friendly role name for user
+    :param role: role as defined in superperms.models
+    :returns: (string) JS role name
+    """
+    roles = {
+        ROLE_OWNER: 'owner',
+        ROLE_VIEWER: 'viewer',
+        ROLE_MEMBER: 'member',
+    }
+    return roles.get(role, 'viewer')
+
+
+def _get_role_from_js(role):
+    """return the OrganizationUser role_level from the JS friendly role name
+
+    :param role: 'member', 'owner', or 'viewer'
+    :returns: int role as defined in superperms.models
+    """
+    roles = {
+        'owner': ROLE_OWNER,
+        'viewer': ROLE_VIEWER,
+        'member': ROLE_MEMBER,
+    }
+    return roles[role]
+
+
+def _get_js_rule_type(data_type):
+    """return the JS friendly data type name for the data data_quality rule
+
+    :param data_type: data data_quality rule data type as defined in data_quality.models
+    :returns: (string) JS data type name
+    """
+    return dict(Rule.DATA_TYPES).get(data_type)
+
+
+def _get_rule_type_from_js(data_type):
+    """return the Rules TYPE from the JS friendly data type
+
+    :param data_type: 'string', 'number', 'date', or 'year'
+    :returns: int data type as defined in data_quality.models
+    """
+    d = {v: k for k, v in dict(Rule.DATA_TYPES).items()}
+    return d.get(data_type)
+
+
+def _get_js_rule_severity(severity):
+    """return the JS friendly severity name for the data data_quality rule
+
+    :param severity: data data_quality rule severity as defined in data_quality.models
+    :returns: (string) JS severity name
+    """
+    return dict(Rule.SEVERITY).get(severity)
+
+
+def _get_severity_from_js(severity):
+    """return the Rules SEVERITY from the JS friendly severity
+
+    :param severity: 'error', or 'warning'
+    :returns: int severity as defined in data_quality.models
+    """
+    d = {v: k for k, v in dict(Rule.SEVERITY).items()}
+    return d.get(severity)
+
+
+class EmailAndIDSerializer(serializers.Serializer):
+    email = serializers.CharField(max_length=100)
+    user_id = serializers.IntegerField()
+
+
+class ListUsersResponseSerializer(serializers.Serializer):
+    users = EmailAndIDSerializer(many=True)
+
+
+class UserSchema(AutoSchemaHelper):
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.manual_fields = {
+            ('GET', 'list'): [],
+            ('POST', 'create'): [
+                self.org_id_field(),
+                self.body_field(
+                    name='New User Fields',
+                    required=True,
+                    description="An object containing meta data for a new user: "
+                                "required - [last_name, role(viewer/owner/member), email]",
```

Comment:
I think role is optional

Reply:
![Screen Shot 2020-05-19 at 7 49 00 AM](https://user-images.githubusercontent.com/31779424/82334443-3b31e280-99a5-11ea-86e6-d9a27e7c2f75.png)
I was just going off of the comments within the action.  If it is documented here as required, should we keep it that way?